extensions/pom.xml

```diff
@@ -116,6 +116,18 @@
             <version>${project.parent.version}</version>
             <scope>test</scope>
         </dependency>
+        <!-- explicit version upgrade to avoid CVE-2022-2047 -->
+        <dependency>
+            <groupId>org.eclipse.jetty</groupId>
+            <artifactId>jetty-util-ajax</artifactId>
+            <version>9.4.48.v20220622</version>
+        </dependency>
+        <!-- explicit version upgrade to avoid CVE-2022-2047 -->
+        <dependency>
+            <groupId>org.eclipse.jetty</groupId>
+            <artifactId>jetty-util</artifactId>
+            <version>9.4.48.v20220622</version>
+        </dependency>
```

Comment:
Doesn't updating the version here https://github.com/hazelcast/hazelcast/blob/master/pom.xml#L1519 fix the issue?

Reply:
It does, I've made change on all branches